### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by georgebisbas**

### CODE REVIEW

File: benchmarks/user/README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ working set). There are multiple scenarios that are worth considering:
   parallel region.  Pinning is typically enforced via environment variables.
 
 Some more information about pinning is available
-[here](https://perf.readthedocs.io/en/latest/system.html).
+[here](https://buildmedia.readthedocs.org/media/pdf/perf/latest/perf.pdf).
```
Comment (FabioLuporini) [inline benchmarks/user/README.md]: While I agree we should replace a broken link, I wonder whether this is really the best we can link to...the doc of another tool

Comment (georgebisbas) [reply benchmarks/user/README.md]: Yes of course. It was a very lazy reaction by my side.
What about this: https://hpc-wiki.info/hpc/Binding/Pinning

Comment (FabioLuporini) [reply benchmarks/user/README.md]: yeah, LGTM

**Comment by FabioLuporini** [comment]
Merged, thanks

**Pull request merged**